dataikuapi/dss/notebook.py

```diff
@@ -27,14 +28,20 @@ def unload(self, session_id=None):
                 raise Exception("Several sessions of the notebook are running, choose one")
             else:
                 session_id = state['activeSessions'][0].get('sessionId', None)
-        return self.client._perform_json("DELETE", "/projects/%s/notebooks/" % self.project_key, params={'notebookName' : self.notebook_name, 'sessionId' : session_id})
+        return self.client._perform_json("DELETE",
+                                         "/projects/%s/jupyter-notebooks/%s/sessions/%s" % (self.project_key, self.notebook_name, session_id))
 
     def get_state(self):
         """
         Get the status of the notebook
         """
-        if self.state is None:
-            self.state = self.client._perform_json("GET", "/projects/%s/notebooks/" % self.project_key, params={'notebookName' : self.notebook_name})
+        notebook_list = self.client._perform_json("GET",
```

Comment:
Should we keep the cache mechanism?
(If you put back the cache, maybe you should add with an optional parameter `clear_cache = false` for when the user know the cache is wrong. Or a `clear_cache()` method)